analyzers/maven/maven_test.go

```diff
@@ -0,0 +1,27 @@
+package maven_test
+
+import (
+	"testing"
+
+	"github.com/fossas/fossa-cli/analyzers/maven"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDiscover(t *testing.T) {
```

Comment:
It should work deterministically. I guess it makes it so that if func `Discover` is changed then this test may need to be updated. Is that your concern?